docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Datasources_Agroal.adoc

```diff
@@ -0,0 +1,238 @@
+[[agroal]]
+= Agroal configuration
```

Comment:
Can you add also note that agroal doesn't support lazy enlistment

Reply:
Sure! Thanks for pointing that out.